### DISCUSSION TITLE
Is it possible to pass the input from .prompt() to a Script User Function?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [SilentVoid13/Templater#723] [Help] [answered]
I want to run Javascript against input while creating my note from a template. I'm running into an error when trying to do the following:

`slug: '<% tp.user.convertTitleToSlug(tp.system.prompt('Title:')) %>'`.

From what I can tell, I am getting the error because the `convertTitleToSlug` function is getting an empty parameter. Is this possible?

**Comment by AB1908** [comment, marked as answer]
You need to `await` the suggester. Look at examples at the bottom of the docs page.

**Reply by CodyBontecou** [reply]
You rock, thank you.
